packages/ts-types/src/narrowing/to.ts

```diff
@@ -39,6 +39,7 @@ export function toAnyJson<T>(value: Nullable<T>, defaultValue: AnyJson): AnyJson
 // underlying function
 export function toAnyJson<T>(value: Nullable<T>, defaultValue?: AnyJson): Optional<AnyJson> {
   try {
+    /* eslint-disable-next-line @typescript-eslint/no-unsafe-return */
```

Comment:
Can't we just coerce the type returned by `JSON.parse()` to be `AnyJson` rather than disable the rule?